Detections/Box/BoxUserRoleChangedToOwner.yaml

```diff
@@ -0,0 +1,38 @@
+id: 174c31c9-22ec-42e5-8226-814391c08200
+name: Box - User role changed to owner
+description: |
+  'Detects when user collaboration role is changed to owner.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: BoxDataConnector
+    dataTypes:
+      - BoxEvents_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+query: |
+  let lbperiod = 60d;

```

Comment:
60d is not recommended, 14d max recommended.

Reply:
updated